internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -170,9 +180,39 @@ func (handler StorageHandler) processAsyncGetRequest(writer http.ResponseWriter,
 	handler.helper.Response(writer, resp, http.StatusOK)
 }
 
-// processGetAsyncRequest is used to handle Async POST Requests
+//addEvent is used to handle the add event Post requests
+func (handler StorageHandler) addEvent(writer http.ResponseWriter, request *http.Request) {
+	readingAPI := "/api/v1/" + event
+	serverIP, readingPort, err := config.GetServerIP(configPath)
+
+	if err != nil {
+		http.Error(writer, "Configuration File Not Found", http.StatusNotFound)
+		return
+	}
+
+	requestURL := handler.helper.MakeTargetURL(serverIP, readingPort, readingAPI)
+	reqBody, err := ioutil.ReadAll(request.Body)
+	if err != nil {
+		http.Error(writer, "Error in getting body", http.StatusBadRequest)
+	}
+	resp, statusCode, err := handler.helper.DoPost(requestURL, reqBody)
+	if err != nil {
+		http.Error(writer, "Error in doing Post request :", statusCode)
+		return
+	}
+	eventid := string(resp)
+	log.Debug(fmt.Sprintf("Event Id is %s", eventid))
```

Comment:
What about the suggestion below?
```suggestion
	log.Debug("Event Id is ", eventid)
```